pkg/xen-tools/patches-4.13.0/08-memlogd-logger.patch

```diff
@@ -39,7 +39,7 @@ index a43c57edad..d0bf636c10 100644
  
  	snprintf(logfile, PATH_MAX-1, "%s/guest-%s%s.log",
  		 log_dir, data, con->log_suffix);
-+        snprintf(logname, PATH_MAX-1, "guest_vm/%s%s",
++        snprintf(logname, PATH_MAX-1, "guest_vm-%s%s",
```

Comment:
I'm just curious - why this change?

Reply:
I guess it needs to be a filename with no '/ and not a pathname.

Reply:
I must be missing something, but looking at the rest of the code -- it simply looks for 'guest_vmX' when it parses that field (for any value of X really)

What am I missing?

Reply:
Except for this case, the other xen tools logs like xl, qemu-dm etc use "-" instead of "/". Wanted everything to look same.

Reply:
Ah! That makes sense then.

Reply:
Btw, please don't forget to update the documentation tho @gkodali-zededa

Reply:
will change LOGGING.md to reflect this change.